cms/models.py

```diff
@@ -823,10 +865,6 @@ class CoursePage(ProductPage):
     """
 
     parent_page_types = ["CourseIndexPage"]
-    subpage_types = [
-        "cms.FlexiblePricingRequestForm",
```

Comment:
> This is a sub_type of both `ProgramPage` and `CoursePage` and both of these classes are children of the `ProductPage` class. That's why I moved it to the parent class i.e. `ProductPage` as I've moved the `CertificatePage` here. If you think this can cause some issues I'll move it back.

https://github.com/mitodl/mitxonline/pull/1072#discussion_r999417879